### PR TITLE
add StackEnsembleSettings entity class

### PR DESCRIPTION
**Pull request by sharma-riti**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    def __init__(
+        self,
+        *,
+        stack_meta_learner_k_wargs: Any = None,
+        stack_meta_learner_train_percentage: float = 0.2,
+        stack_meta_learner_type: StackMetaLearnerType = None,
+        **kwargs
+    ):
+        """
+        :keyword stack_meta_learner_k_wargs: Optional parameters to pass to the initializer of the
```
Comment (novaturient95) [inline sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py:23]: should we use param and key here, to be consistent with rest of v2?

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    StackEnsembleSettings as RestStackEnsembleSettings,
+    StackMetaLearnerType
+)
+
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
+
+class StackEnsembleSettings(RestTranslatableMixin):
+    """Advances setting to customize StackEnsemble run.
```
Comment (novaturient95) [inline sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py:11]: nit: Advance

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    """
+
+    def __init__(
+        self,
+        *,
+        stack_meta_learner_k_wargs: Any = None,
```
Comment (novaturient95) [inline sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py:17]: nit: kwargs

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py (hunk 1 above)
Comment (sharma-riti) [reply sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py]: fixed.

File: sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py (hunk 3 above)
Comment (sharma-riti) [reply sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py]: maybe I can leave this one as it is for now to align with rest contracts... ?

**Review by novaturient95 — APPROVED** [review]

**Review by singankit — APPROVED** [review]

**Review by singankit — APPROVED** [review]

**Review by svaruag — APPROVED** [review]

**Pull request merged**